omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,243 @@
+                                            <controls>
+                                                <when value="90002" thenDisplay="#other-regimen"/>
+                                            </controls>
+                                            </obs>
+                                        <label>                                            
+                                            Parent Health facility
+                                        </label>
+                                        <obs id="parent-health-facility" conceptId="166453" required="true"/>
+                                        <obs id="district-of-parent-health-facility" labelText="District of Parent Health Facility" conceptId="166454" />
+                                        <label>                                            
+                                            Serial No
+                                        </label>
+                                        <obs id="serial-number" conceptId="1646" required="true"/>                                        
+                                    </div>
+
+                                    <div class="row">
+                                        <div class="col-md-12">
+                                            <obs id="reason-for-visit-other" labelText="Other Specify" conceptId="164975" />
+                                            <obs id="other-regimen" labelText="Other Regimen" conceptId="99126" />
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header">
+                                Patient Demographics
+                            </div>
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-4">
+                                        <label>Phone Number:</label>
+                                        <lookup expression="personAttributes.get('Telephone Number')"/>
+                                        <label>ART Number at Parent Facility</label>
+                                        <span id="patientNumber">
```

Comment:
@ODORA0 The Patient Clinic number should be an Obs not an identifier as it may conflict with existing ART numbers

Reply:
@ssmusoke Done

Reply:
@ODORA0 This has not been updated